### ISSUE TITLE
Errors appear in sonar-scanner log

### ISSUE THREAD

**Issue by begarco** [lequal/sonar-icode-cnes-plugin#11]
### Expected behavior
Sonar-scanner execution must be free of errors.

### Actual behavior
Some errors appear during sonar-scanner analysis: 
```ERROR: Unexpected report analysis end element:analysisInformations
ERROR: Unexpected report analysis end element:analysisFile
ERROR: Unexpected report analysis end element:result
ERROR: Unexpected report analysis end element:result
ERROR: Unexpected report analysis end element:result
ERROR: Unexpected report analysis end element:result
```

### Steps to reproduce behavior
* Set following settings:
  * extension for result files: .res
  * source files' suffixes : \*.f,\*.f77,\*.f90,\*.sh,\*.F,\*.F77,\*.F90
  * results folder : .
* Create a quality profile containing all rules and set it as default
* Execute sonar scanner in APSQ-005 : [APSQ-005.zip](https://github.com/lequal/sonar-icode-cnes-plugin/files/1834002/APSQ-005.zip)

### Detection version
* Plugin version: V0
* SonarQube version: 6.7.1
